java/client/src/main/java/glide/api/models/commands/LinsertOptions.java

```diff
@@ -0,0 +1,15 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.ListBaseCommands;
+
+/** Options for {@link ListBaseCommands#linsert}. */
+public class LinsertOptions {
```

Comment:
Just curious, what's the advantage of embedding InsertPosition inside LinsertOptions instead of only having InsertPosition in this file (ie removing the outer class)?

Reply:
No advantage in general. May be naming?
`LinsertOptions.InsertPosition.AFTER`